kazoo/recipe/lock.py

```diff
@@ -555,6 +566,10 @@ def _inner_release(self):
         self.is_acquired = False
         return True
 
+    def break_lock(self):
```

Comment:
No worries I can see about the semaphore part, it's based on leases so I'm
pretty sure breaking it is slightly more complicated.

On Sunday, December 6, 2015, SpamapS <notifications@github.com> wrote:

> In kazoo/recipe/lock.py
> <https://github.com/python-zk/kazoo/pull/371#discussion_r46773149>:
>
> > @@ -555,6 +566,10 @@ def _inner_release(self):
> >          self.is_acquired = False
> >          return True
> >
> > +    def break_lock(self):
>
> I am flying a bit blind here. This works for the case I have for tooz.
> Beyond that, I'm entirely open to suggestions.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/python-zk/kazoo/pull/371/files#r46773149>.
>


-- 
--
facebook.com/jshharlow <http://www.facebook.com/jshharlow>
flickr.com/jshharlow
YIM: jshharlow